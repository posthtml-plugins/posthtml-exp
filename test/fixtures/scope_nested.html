<p>{{key}}</p>
<scope with="scope">
  <p>{{key}}</p>
  <scope with="one">
    <p>{{key}}</p>
  </scope>
  <scope with="two">
    <p>{{key}}</p>
  </scope>
</scope>
<p>{{key}}</p>
