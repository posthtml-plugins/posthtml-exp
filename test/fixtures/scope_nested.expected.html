<p>global</p>

  <p>scope</p>

    <p>one</p>


    <p>two</p>


<p>global</p>
